server/internal/handler/eatery.go

```diff
@@ -20,10 +21,28 @@ func (h *Handler) GetEateries(c echo.Context, params schema.GetEateriesParams) e
 
 // PostEateries implements schema.ServerInterface.
 func (h *Handler) PostEateries(c echo.Context, params schema.PostEateriesParams) error {
-	return c.JSON(http.StatusNotImplemented, schema.Error{
-		Code:  "NOT_IMPLEMENTED",
-		Error: "PostEateries endpoint is not implemented yet",
-	})
+	var req schema.EateryCreate
+	if err := c.Bind(&req); err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest, "bad request")
+	}
+
+	createParams := repository.CreateEateryParams{
+		Name:        req.Name,
+		Description: req.Description,
+	}
+	eateryID, err := h.repo.CreateEateries(c.Request().Context(), createParams)
+
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, "fail to create eatery")
+	}
+
+	res := schema.Eatery{
+		Id:          eateryID,
+		Name:        req.Name,
+		Description: req.Description,
+	}
+
+	return c.JSON(http.StatusBadRequest, res)
```

Comment:
もう1点だけ！
Statusはリクエストが正常に処理されたかを表すものなので、成功したときは`StatusOK`を返してほしいです！
```suggestion
	return c.JSON(http.StatusOK, res)
```